tools/keytools/keygen.c

```diff
@@ -34,6 +34,7 @@
 #include <sys/stat.h>
 #include <sys/types.h>
 #include <errno.h>
+#include <libgen.h>
```

Comment:
Note... we also have a python version of the keygen tool, which may also need to be updated.